src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/IPaymentsPublisher.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus;
+
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.PaymentsData;
+
+public interface IPaymentsPublisher {
```

Comment:
do we need this interface?

Reply:
It is used in functional tests similar to IProcessedEnvelopeNotifier